### PR TITLE
[REVIEW] Feature/docker

### PR DESCRIPTION
**Pull request by rosscdh**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+RUN python setup.py install
+
+# uWSGI will listen on this port
+EXPOSE 8000
+
+# # Call collectstatic (customize the following line with the minimal environment variables needed for manage.py to run):
+# RUN DATABASE_URL=none /venv/bin/python manage.py collectstatic --noinput
+
+# Start uWSGI
+ENTRYPOINT ["coderedcms"]
```
Comment (vsalvino) [inline Dockerfile]: Question... running the `coderedcms` command will not actually do anything by default. It is used as `coderedcms start` to create a new boilerplate project. So is this the most appropriate entrypoint command? I'm not really sure what the purpose of this docker image is.

Comment (rosscdh) [reply Dockerfile]: yes this is true. I suspect i may have miudnerstood the purpose of this repo. but regardless an interesting project.

Comment (rosscdh) [reply Dockerfile]: also using `coderedcms` as the entrypoint allows you to do things like `docker run --rm -v $PWD/clients/myclient:/src/output coderedcms:latest start`

Comment (rosscdh) [reply Dockerfile]: ps yes it is the valid entrypoint; allows you to run start in your docker compose or kube config

Comment (rosscdh) [reply Dockerfile]: `docker run --rm -v $PWD/clients/myclient:/src/output coderedcms:latest start my_project` which will output the generated to `/src/output`

**Pull request closed without merging**